src/components/gds-heading/_index.scss

```diff
@@ -1,6 +1,8 @@
+@use "../../styles/typography" as *;
```

Comment:
I dont think you need the `as *` part.

Reply:
Thanks, I haven't used this syntax that much, so good to know when I'm  writing something unwanted. :)